packages/intrepid2/src/Orientation/Intrepid2_OrientationTools.hpp

```diff
@@ -98,8 +98,9 @@
 #include "Intrepid2_HDIV_TET_I1_FEM.hpp"
 #include "Intrepid2_HDIV_WEDGE_I1_FEM.hpp"
 
-#include "Intrepid2_HierarchicalBasisFamily.hpp"
-#include "Intrepid2_NodalBasisFamily.hpp"
+/// KK: these cannot be included now
+//#include "Intrepid2_HierarchicalBasisFamily.hpp"
```

Comment:
@kyungjoo-kim we cannot do this. We are not in such a rush. Roger is working on including projections in Empire. There are tests using orientations and projections for hierarchical basis

Reply:
I can include it but I would not test it.

Reply:
the headers are included but it does not effect anything. I don't understand why this is included in orientation tools if they are not used.

Reply:
You are probably right. Maybe it was there before we started using `getReferenceSubCell` and it's no longer needed now.